directories/src/redux/services/broker-node.js

```diff
@@ -2,38 +2,99 @@ import axios from "axios";
 import { API_HEADERS, API_ROOT_URL, API_REQUEST_ERROR } from "../../config/";
 
 const registerWebnode = address =>
-  axios({
-    method: "POST",
-    url: `${API_ROOT_URL}/api/v2/supply/webnodes`,
-    data: { address }
```

Comment:
keep it like this. instead of wrapping it in a promise, let the epic handle the error

Reply:
ok, understand